views/form.erb

```diff
@@ -48,6 +48,18 @@
             <input type="url" id="application_webhook_url" name="application[webhook_url]" class="form-control" placeholder="https://example.org/webhook_handler" value="<%= @application.webhook_url %>" >
           </div>
 
+          <div class="form-group <%= @application.errors.on(:legislature) ? 'has-error' : '' %>">
+            <label class="control-label" for="application_legislature">Legislature</label>
+            <select id="application_legislature" name="application[legislature]" class="form-control">
+                <% @countries.each do |country| %>
+                    <option value="">All legislatures</option>
+                    <% country.legislatures.each do |legislature| %>
+                        <option value="<%= country.name %>/<%= legislature.name %>"><%= country.name %> - <%= legislature.name %></option>
```

Comment:
Having thought about it, a quick way to make this slightly less hacky is to use the `legislature_for_file` method that's been added in this pull request. This would mean that we're at least being consistent in how we extract the `Country/Legislature` bit from a path:

```erb
<option value="<%= legislature_for_file(legislature.raw_data[:sources_directory]) %>">
```

This feels like something that would perhaps be useful if we added it to [everypolitician-ruby](https://github.com/everypolitician/everypolitician-ruby), maybe something like `legislature.directory`?